src/2_resources.js

```diff
@@ -334,6 +335,23 @@ resources.link = {
 	})
 };
 
+resources.deepview = {
+	destination: config.api_endpoint,
+	endpoint: "/v1/deepview",
+	jsonp: true,
+	method: utils.httpMethod.POST,
+	params: defaults({
+		"campaign": validator(false, validationTypes.STRING),
+		"channel": validator(false, validationTypes.STRING),
+		"data": validator(true, validationTypes.STRING),
```

Comment:
actually no - it looks like it's sending back stringified json - in utils.cleanLinkData, `linkData['data'] = goog.json.serialize(linkData['data'] || {});`